File: remove_blurs.py
import re

with open('index.html', 'r', encoding='utf-8') as f:
    content = f.read()

# Remove full CSS property lines for filter and backdrop-filter
content = re.sub(r'^\s*(?:-webkit-)?backdrop-filter:\s*blur\([^)]+\);?\s*$\n?', '', content, flags=re.MULTILINE)
content = re.sub(r'^\s*filter:\s*blur\([^)]+\);?\s*$\n?', '', content, flags=re.MULTILINE)

# Remove inline backdrop-filter: blur(...);
content = re.sub(r'(?:-webkit-)?backdrop-filter:\s*blur\([^)]+\);?', '', content)
content = re.sub(r'filter:\s*blur\([^)]+\);?', '', content)

# Remove filter: "blur(...)" from JS objects
content = re.sub(r',\s*filter:\s*[\'"]blur\([^)]+\)[\'"]', '', content)
content = re.sub(r'filter:\s*[\'"]blur\([^)]+\)[\'"]\s*,', '', content)
content = re.sub(r'filter:\s*[\'"]blur\([^)]+\)[\'"]', '', content)

# Handle the complex scroll blur logic in GSAP around line 3043
content = re.sub(r'filter:\s*`blur\(\$\{blur\}px\)`', '/* removed blur */', content)

with open('index.html', 'w', encoding='utf-8') as f:
    f.write(content)

print("Blurs removed successfully.")

File: remove_blurs.js
const fs = require('fs');

let content = fs.readFileSync('index.html', 'utf8');

// Remove full CSS property lines for filter and backdrop-filter
content = content.replace(/^\s*(?:-webkit-)?backdrop-filter:\s*blur\([^)]+\);?\s*$/gm, '');
content = content.replace(/^\s*filter:\s*blur\([^)]+\);?\s*$/gm, '');

// Remove inline backdrop-filter: blur(...);
content = content.replace(/backdrop-filter:\s*blur\([^)]+\);?/g, '');
content = content.replace(/-webkit-backdrop-filter:\s*blur\([^)]+\);?/g, '');
content = content.replace(/filter:\s*blur\([^)]+\);?/g, '');

// Remove filter: "blur(...)" from JS objects
content = content.replace(/,\s*filter:\s*['"]blur\([^)]+\)['"]/g, '');
content = content.replace(/filter:\s*['"]blur\([^)]+\)['"]\s*,/g, '');
content = content.replace(/filter:\s*['"]blur\([^)]+\)['"]/g, '');

// Handle the complex scroll blur logic in GSAP around line 3043
// This was likely a custom cursor or scroll effect that had blur. 
// We'll replace filter: `blur(${blur}px)` with nothing.
content = content.replace(/filter:\s*`blur\(\$\{blur\}px\)`/g, '/* removed blur */');

fs.writeFileSync('index.html', content, 'utf8');
console.log("Blurs removed successfully.");
